package.json

```diff
@@ -67,9 +69,11 @@
     "typescript": "~2.3.4"
   },
   "peerDependencies": {
-    "lodash": "4.x"
+    "lodash": "4.x",
+    "tslib": "^1.7.1"
   },
   "dependencies": {
-    "tslib": "^1.6.1"
+    "lodash": "^4.17.4",
```

Comment:
How is this going to resolve if my application is using lodash v4.3? Is it going to use v4.3 for the decorators or is it going to install version 4.17.4 along with it?